project-keeper/src/main/java/com/exasol/projectkeeper/validators/files/FileTemplatesFactory.java

```diff
@@ -85,6 +94,9 @@ private List<FileTemplate> getMavenTemplates(final AnalyzedMavenSource source) {
         if (enabledModules.contains(LOMBOK)) {
             templates.add(new FileTemplateFromResource("lombok.config", REQUIRE_EXACT));
         }
+        if (enabledModules.contains(NATIVE_IMAGE)) {
+            templates.add(new EmptyFileTemplate("src/main/reflect-config.json", REQUIRE_EXIST));
```

Comment:
An empty file is not valid json. Could you use `{}` as default file content?

Reply:
Good point. I chenged it.